bfsscp/bfsscp_test.go

```diff
@@ -31,11 +32,34 @@ var _ = Describe("Bucket", func() {
 	})
 
 	It("should register scp scheme", func() {
-		subject, err := bfs.Connect(context.Background(), "scp://root:root@127.0.0.1:7022/prefix?tmpdir=test")
+		subject, err := bfs.Connect(ctx, "scp://root:root@127.0.0.1:7022/~/prefix?tmpdir=test")
 		Expect(err).NotTo(HaveOccurred())
 		Expect(subject.Close()).To(Succeed())
 	})
 
+	It("should allow for absolute and relative paths", func() {
+		subject, err := bfs.Connect(ctx, "scp://root:root@127.0.0.1:7022/~/prefix")
+		Expect(err).NotTo(HaveOccurred())
+
+		f, err := subject.Create(ctx, "test", nil)
+		Expect(err).NotTo(HaveOccurred())
+		Expect(f.Commit()).To(Succeed())
+
+		i, err := subject.Head(ctx, "test")
+		Expect(err).NotTo(HaveOccurred())
+		Expect(i).NotTo(BeNil())
+
+		Expect(subject.Close()).To(Succeed())
+
+		subject, err = bfs.Connect(ctx, "scp://root:root@127.0.0.1:7022/prefix")
+		Expect(err).NotTo(HaveOccurred())
+		i, err = subject.Head(ctx, "test")
```

Comment:
Nit: I'd insert a newline above and be more verbose with naming (`i` -> `info`), but that's really minor.